src/hooks/useTrackEvent.ts

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { useChrome } from '@redhat-cloud-services/frontend-components/useChrome';
+
+type JSONValue = string | number | boolean | JSONObject | Array<JSONValue>;
+
+interface JSONObject {
+  [x: string]: JSONValue;
+}
+
+export type AnalyticProps = {
+  event: string;
+  properties?: JSONObject;
+};
+
+export const useTrackEvent = () => {
+  const { analytics } = useChrome();
+  return React.useCallback(
+    (event: string, properties?: JSONObject) => {
+      if (process.env.NODE_ENV !== 'development') {
+        analytics?.track(event, { current_path: window.location.pathname, ...properties });
```

Comment:
We have both dev and prod segment instances. Unless you are developing against the prod environment, you should not need this.

Reply:
@Hyperkid123 I'm assuming that we use a production build for `prod` and `stage`. The check for `development` is to not send events while developers are running a local dev environment.

Reply:
Got it. We can keep this. But just FYI, the do actually filter out RH accounts in the analytics tool (at leas in Pendo). We should not get any noise from developers if that is what you are worried about
.